src/internal_events/aws_ecs_metrics.rs

```diff
@@ -0,0 +1,115 @@
+use super::InternalEvent;
+use metrics::{counter, histogram};
+use std::borrow::Cow;
+use std::time::Instant;
+
+#[derive(Debug)]
+pub struct AwsEcsMetricsReceived {
+    pub byte_size: usize,
+    pub count: usize,
+}
+
+impl InternalEvent for AwsEcsMetricsReceived {
+    fn emit_logs(&self) {
+        debug!(message = "Scraped events.", ?self.count);
+    }
+
+    fn emit_metrics(&self) {
+        counter!(
+            "events_processed", self.count as u64,
```

Comment:
We've updated all our metrics to basically use the Prometheus style metric naming (see https://github.com/timberio/vector/pull/4647).